apps/src/music/settings.ts

```diff
@@ -50,16 +50,12 @@ export function useMusicSettings(): Setting[] {
     setSelectedTheme(name);
   };
 
-  if (selectedTheme === null) {
-    return []; // still loading
-  }
-
   return [
     {
       id: 'blocklyTheme',
       label: commonI18n.blocklyTheme(),
       options: blockThemeOptions,
-      selectedValue: selectedTheme,
+      selectedValue: selectedTheme || undefined,
```

Comment:
if the selected value is undefined do we fall back to a certain option in the list?

Reply:
The first option is selected if it's undefined. For this setting that means the default theme.